src/Compilers/CSharp/Test/Syntax/Parsing/ParserRegressionTests.cs

```diff
@@ -79,28 +77,13 @@ public void c024928()
             var tree = SyntaxFactory.ParseSyntaxTree(test);
         }
 
+#if !DEBUG 
```

Comment:
My reasoning for wanting to use a custom `Fact` is the following:

1. In general I dislike `#if` directives and want to minimize them because it makes it harder to accurately analyze source code as it creates multiple views of the code.  This is a frequent problem in tools like `codeformatter`.  If we run this tool with `Release` defined (which is a valid idea) then it will never process any of this added code.  
2. Our pattern thus far has been to conditionally execute tests by using a custom `Fact` attribute.  Using `#if` manually here goes against that pattern.